src/bootloaders/systemd-class.c

```diff
@@ -313,6 +317,17 @@ bool sd_class_install_kernel(const BootManager *manager, const Kernel *kernel)
         if (root_dev->btrfs_sub) {
                 cbm_writer_append_printf(writer, "rootflags=subvol=%s ", root_dev->btrfs_sub);
         }
+        /* Add Btrfs information if relevant */
+        if (root_dev->btrfs_sub) {
+                cbm_writer_append_printf(writer, "rootflags=subvol=%s ", root_dev->btrfs_sub);
+        }
```

Comment:
Isn't this already present in the lines just above this?

Reply:
Yeah, must've pressed paste one too many times :see_no_evil: